app/views/posts/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>List of post by a user</h1>
```

Comment:
- [ ]  You should have corresponding view for show method from postController as well.  This will help your show method to display the information of specific post in the browser.